ros2_generate_interface_docs/ros2_generate_interface_docs/templates/msg.html.em

```diff
@@ -19,7 +19,7 @@ import time
 @[for line in raw_text.splitlines()]@
 @{  text = line.strip('#')}@
 @[  if '#' in line]@
```

Comment:
Does the whole line get the same style if there is an inline comment?
```
int field # Some inline comment
```

Reply:
It's on my TODO list to review all the interface to avoid inline comments